app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -111,7 +111,11 @@ export function OceanInterface (): JSX.Element | null {
       })
       broadcastTransaction(transaction.tx, client)
         .then(async () => {
-          setTxUrl(getTransactionUrl(transaction.tx.txId))
+          try {
+            setTxUrl(getTransactionUrl(transaction.tx.txId))
```

Comment:
Move this out, `transaction.tx` is already signed (unlike previous flow), `txId` is guaranteed there.